### DISCUSSION TITLE
Ability to filter feed?

### DISCUSSION THREAD

**Discussion by mccordgh** [community/community#16419] [Other Feature Feedback, Questions, & Ideas]
The ability to filter the feed would be useful (say to only see created repos, etc). Right now I don't really care to look at my feed because its a bunch of starred repos that I don't necessarily care about. I would rather see what is being created on their end and other things :)

**Comment by siminapasat** [comment]
Hi @mccordgh 👋 Apologies for the late reply. We have this functionality coming very soon!

**Comment by tnir** [comment]
https://github.blog/changelog/2022-06-14-updates-to-github-feed-public-beta/ is still not enough to me...

**Reply by siminapasat** [reply]
Can you please expand?

**Reply by tnir** [reply]
> Recommended based on people you follow

was what I do not need. It looks like my ex-coworkers 😭 , but I still want some Recommendations from GitHub.

I already filter Stars and Follows not to include in my feed. These filters made me soooo happy 😍 .

**Reply by sfkpmr** [reply]
Yep, I would use it if I could set it to only show announcements and releases from repos I watch/star. 

Right now? "You're seeing this because you collaborated with blabla-bot", come on ...

**Reply by Codel1417** [reply]
yeah the filtering is lacking quite a bit. Cant hide `Recommended based on people you follow`, completely.

**Comment by jafludo** [comment]
Hey !

**Comment by SonniRoy** [comment]
Sangat berarti bagi saya untuk membuat kita menjadi orang mempunyai pola pikir sendiri 😇

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by Codel1417** [reply]
This was out for awhile but doesn't offer a way to view recommended repositories without viewing people suggestions

**Comment by ris58h** [comment]
It doesn't work for me https://github.com/orgs/community/discussions/28987
